vue/sbc-common-components/src/components/NotificationPanel.vue

```diff
@@ -0,0 +1,175 @@
+                  <span :class="!item.read && (item.priority ? 'dot-red' : 'dot-blue')">
+                  </span>
+                </v-col>
+                <v-col>
+                <v-list-item-content>
+                  <v-list-item-title class="font-weight-bold list-subtitle">{{item.title}}</v-list-item-title>
+                  <v-list-item-subtitle>{{item.date}}</v-list-item-subtitle>
+                  <v-spacer></v-spacer>
+                  <v-list-item-content v-html="item.description"></v-list-item-content>
+                </v-list-item-content>
+                </v-col>
+              </v-row>
+            </v-list-item>
+            <v-divider v-if="i < notifications.length - 1" :key="`${i}-divider`"></v-divider>
+          </template>
+        </v-list-item-group>
+      </v-list>
+    </v-navigation-drawer>
+
+  </div>
+</template>
+
+<script lang='ts'>
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { Notification } from '../models/notification'
+import { mapState, mapActions } from 'vuex'
+import NotificationModule from '../store/modules/notification'
+import { getModule } from 'vuex-module-decorators'
+import 'clickout-event'
+
+@Component({
+  name: 'NotificationPanel',
+  beforeCreate () {
+    this.$store.isModuleRegistered = function (aPath: string[]) {
+      let m = (this as any)._modules.root
+      return aPath.every((p) => {
+        m = m._children[p]
+        return m
+      })
+    }
+    if (!this.$store.isModuleRegistered(['notification'])) {
+      this.$store.registerModule('notification', NotificationModule)
+    }
+    this.$options.computed = {
+      ...(this.$options.computed || {}),
+      ...mapState('notification', ['notifications'])
+    }
+    this.$options.methods = {
+      ...(this.$options.methods || {}),
+      ...mapActions('notification', ['markAsRead'])
+    }
+  }
+})
+export default class NotificationPanel extends Vue {
+  private readonly notifications!: Notification[]
+
+  /** Prop to display the dialog. */
+  @Prop() showNotifications: boolean
+
+  @Emit('closeNotifications')
+  private async emitClose () {
+
+  }
+
+  private async mounted () {
+    getModule(NotificationModule, this.$store)
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../assets/scss/theme.scss";
+
+::v-deep ::-webkit-scrollbar {
+  width: 2px;
+}
+
+::v-deep ::-webkit-scrollbar-thumb {
+  background: black;
+  border-radius: 20px;
+}
+
+.navigation-drawer {
+  top: 100px;
+  max-height: calc(100% - 0%);
+}
+
+.dialog-close {
+  position: absolute;
+  top: 8px;
+  right: 15px;
+  margin-right: 70px;
+  z-index: 2;
+  font-weight: bold;
+}
+
+::v-deep .v-btn:not(.dialog-close) .v-icon.v-icon {
+  font-size: 1.25rem !important;
+}
+
+::v-deep .v-btn__content {
+  line-height: inherit;
+}
+
+.toolbar-title {
+  color: $BCgovBlue5;
+  font-size: 1.125rem;
+  font-weight: 700;
+}
+
+.list-subtitle {
+  font-size: 1.125rem;
+  font-weight: 700;
```

Comment:
Since we don't have several levels of bolding, some of us have been using just 'normal' and 'bold' for font weights. In fact, you use 'bold' on line 128 above.